install-build-deps

```diff
@@ -34,6 +34,7 @@ inst_Debian() {
         pkg-config              \
         python3                 \
         python3-dev             \
+        xxd                     \
```

Comment:
Hey, @dwightguth, could you please add it to this section of [README.md](https://github.com/runtimeverification/k/blob/master/README.md#the-long-version)